index.html

```diff
@@ -106,7 +107,7 @@ <h2>Leave a Message</h2>
         </form>
     </section>
 
-    <section id="messages">
+    <section id="Message">
```

Comment:
This id was also used in your js file and it's broke now. Be sure to update the id there, too.